lib/MusicBrainz/Server/EditSearch/Query.pm

```diff
@@ -67,7 +67,7 @@ has combinator => (
 );
 
 has order => (
-    isa => enum([qw( asc desc rand )]),
+    isa => enum([qw( asc desc closed-asc closed-desc voteclosing-asc voteclosing-desc rand )]),
```

Comment:
Every other field/value I can see in the edit search uses underscores instead of hyphens to separate words, so I'd prefer that for consistency. And if we're doing that for readability, we might as well separate vote closing too (`vote_closing_asc`).